v2/emailpassword/advanced-customizations/apis-override/usage.mdx

```diff
@@ -9,7 +9,9 @@ import TabItem from '@theme/TabItem';
 
 # How to use
 
-## Use the override config:
+## Method 1: Use the override config:
+
+If you would like to change the input or ouput specifications for an API or would like to send analytics, you should use this method.
```

Comment:
- Typo on "ouput" -> figure out an extension to use to detect spelling errors.
- The reason you have written to use Method 1 makes no sense.... Change it to "If you would like to change something per or post our API logic, then use this method."